client/src/pages/ForgotPasswordPage/styles.module.scss

```diff
@@ -0,0 +1,128 @@
+@import '../../consts/variables.scss';
+
+.reset{
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  &__title {
+    width: 100%;
+    height: 15%;
+    margin-top: 4rem;
+    text-align: center;
+    & > h1 {
+      text-transform: uppercase;
+      color: $purple;
+      margin: auto;
+      height: 70px;
+      width: 341px;
+      top: 187px;
+      font-weight: bold;
+      border-radius: nullpx;
+      font-size: 48px;
+      margin-bottom: -10px;
+    }
+    & > p {
+      text-transform: uppercase;
+      font-weight: bold;
+      font-size: 18px;
+      margin-top: 0;
+    }
+  }
+
+.mainBox {
+  width: 600px;
+  height: 500px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  .dataInputFields {
+      background-color: #fff;
+      height: 100%;
+      width: 100%;
+      margin: auto;
+      border-radius: 50px;
```

Comment:
10px

Reply:
can you explain more precisely what is wrong with the radius?